soc/arc/snps_emsdp/soc.c

```diff
@@ -0,0 +1,12 @@
+
+/*
+ * Copyright (c) 2019 Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
if file is empty, why have it?

Reply:
to make code organization structure uniformed with other arc core board.

Reply:
we should remove the file.

Reply:
removed